src/core/client/stream/tabs/Comments/Comment/flattenReplies/index.ts

```diff
@@ -7,6 +7,6 @@ export const isReplyFlattened = (
   return (
     flattenedRepliesEnabled &&
     indentLevel &&
-    indentLevel >= FlattenRepliesIndentLevel
+    indentLevel >= MAX_REPLY_INDENT_DEPTH
```

Comment:
It looks like the collapse button is still disappearing one level too soon. I think this is because before the constant was 4 for 4 levels, and now it is 7 for 8 levels. If you change this to `indentLevel > MAX_REPLY_INDENT_DEPTH`, then I think it will show the collapse button for all expected levels.

Reply:
Agreed, I think we have an off-by-one error here! 😆

Reply:
✅